src/components/FilterInput.svelte

```diff
@@ -4,6 +4,8 @@
   export let onChangeText;
   export let filterText = "";
 
+  export let text;
```

Comment:
I think placeHolder might be a better name here?

```suggestion
  export let placeHolder;
```